src/vs/platform/environment/common/argv.ts

```diff
@@ -7,7 +7,15 @@
  * A list of command line arguments we support natively.
  */
 export interface NativeParsedArgs {
-	_: string[];
+	/**
```

Comment:
Where is the args object being constructed? It is not very clear to me how adding properties to this interface will fix the issue.

Reply:
They come from here ([line 72](https://github.com/microsoft/azuredatastudio/pull/23312/files#diff-6f7ec3ad80fb1c5586de9fb0f2c44e28bc02f6204472fa7cb5a0175870cd4b46R72))

![image](https://github.com/microsoft/azuredatastudio/assets/13396919/3c779704-f09e-4578-ab6b-2a00e88bd52b)